hathor/p2p/sync_v2/agent.py

```diff
@@ -332,37 +346,43 @@ def run_sync_transactions(self) -> None:
                       end_block_height=block_height)
         self.send_get_transactions_bfs(needed_txs, block.hash)
 
+    def get_my_best_block(self) -> BlockInfo:
+        """Return my best block info."""
+        bestblock = self.tx_storage.get_best_block()
+        assert bestblock.hash is not None
+        meta = bestblock.get_metadata()
+        # assert not meta.voided_by
```

Comment:
Remove or uncomment this?

Reply:
I'll leave a comment. I think this assert should be there but the current implementation does not work with it.

Reply:
I think the assertion is true when there is no tie, otherwise it's possible that the block returned by `get_best_block` becomes voided when the block with a tie is added (because the previous block won't be removed from the height index).

Reply:
I agree that we can remove it for now. Done! f880a62fac1873f7d8abb384f6209d1b919e6e46